### ISSUE TITLE
Failure to list rooms after succesful login

### ISSUE THREAD

**Issue by ifitzpat** [alphapapa/ement.el#181]
Hi @alphapapa  

Thanks for your great work! I'm having some problems getting ement.el to work. My *Messages* buffer lists the following:

```
Ement: Response arrived after 10.08 seconds.  Reading 1.4k JSON response...
Ement: Reading JSON took 0.00 seconds
Ement: Reading events... 
Ement: Sync request sent, waiting for response...
Invalid face attribute :family nil [2 times]
error in process sentinel: ement-room-list-column-format-notifications: Args out of range: [], 0
```

I'm on Emacs version 29.0.91 and element version 0.10

Thanks!

**Comment by alphapapa** [comment]
Hello @ifitzpat,

First, please note that this client is named Ement.el or Ement.  Element is at https://app.element.io, the official client.

Also, please note that Emacs 29 is not yet released.

Regarding the errors you shared: please try to reproduce the problem in a clean Emacs configuration, e.g. using https://github.com/alphapapa/with-emacs.sh.

Also, please tell me how many rooms your account is in.

Thanks.

**Comment by ifitzpat** [comment]
Thanks for your quick reply! I'm sorry I, of course, meant Ement (not Element). I also realise Emacs 29 isn't released yet and some bugs should be expected. I can verify the the problem also occurs in a clean Emacs install. My account is in 9 rooms. From the traceback it looks like the error occurs in `ement-room-notification-state`  possibly in the `tweak-rule-p` part.

**Comment by alphapapa** [comment]
Would you provide the backtrace then, please?

**Comment by alphapapa** [comment]
This should now be fixed on master and be in the next version, 0.11.  Thanks, @ifitzpat and @Stebalien.
